packages/x-date-pickers/src/internals/demo/DemoContainer.tsx

```diff
@@ -160,7 +160,7 @@ export function DemoContainer(props: DemoGridProps) {
       };
     }
   } else if (childrenSupportedSections.has('date-time')) {
-    sx = { ...sx, [`& > .${textFieldClasses.root}`]: { minWidth: 270 } };
+    sx = { ...sx, [`& .${textFieldClasses.root}`]: { minWidth: 270 } };
```

Comment:
This did not seem to be working, or at least working all the time when we were excpecting it to work.
This change caused all the Argos diffs.

Reply:
Since it's a one liner, would you care extracting it to a standalone PR to merge it quickly and have no Argos diff on this PR
That way if you cause a regression it will be easier to spot :+1: